rpcserver.go

```diff
@@ -7227,13 +7227,28 @@ func (r *rpcServer) SendPayment(req *tchrpc.SendPaymentRequest,
 	pReq := req.PaymentRequest
 	ctx := stream.Context()
 
-	// Do some preliminary checks on the asset ID and make sure we have any
-	// balance for that asset.
-	if len(req.AssetId) != sha256.Size {
-		return fmt.Errorf("asset ID must be 32 bytes")
+	var specifier asset.Specifier
+
+	switch {
+	case len(req.AssetId) > 0 && len(req.GroupKey) > 0:
```

Comment:
Could probably use `parseAssetSpecifier` here? Combined with `asset.NewSpecifier()`. Or maybe even `specifierWithGroupKeyLookup` if that makes sense here?